src/main/webapp/assets/feedStyle.css

```diff
@@ -86,6 +86,7 @@
   padding: 10px;
   padding-right: 30px;
   padding-left: 30px;
+  transition: box-shadow 0.5s;
```

Comment:
Cool! Are you planning on having the shadow hold?

Reply:
I tried it out, and I decided that I liked the look of the 'brief flare' better :)